DQM/L1TMonitor/src/L1TStage2uGT.cc

```diff
@@ -20,9 +20,17 @@ L1TStage2uGT::L1TStage2uGT(const edm::ParameterSet& params):
    algoBitFirstBxInTrain_(-1),
    algoBitLastBxInTrain_(-1),
    algoNameFirstBxInTrain_(params.getUntrackedParameter<std::string>("firstBXInTrainAlgo","")),
-   algoNameLastBxInTrain_(params.getUntrackedParameter<std::string>("lastBXInTrainAlgo",""))
+   algoNameLastBxInTrain_(params.getUntrackedParameter<std::string>("lastBXInTrainAlgo","")),
+   algoBitUnpre_(-1),
+   algoBitDis_(-1),
+   unprescaledAlgoShortList_(params.getUntrackedParameter<std::vector<std::string>> ("unprescaledAlgoShortList")),
+   disabledAlgoShortList_(params.getUntrackedParameter<std::vector<std::string>> ("disabledAlgoShortList")),
+   range_(5)
```

Comment:
This variable is not needed. There is no need to reserve space anymore for the vectors once they are initialised in the initializer list.